src/features/payment/PaymentScreen.tsx

```diff
@@ -351,25 +348,17 @@ const PaymentScreen = () => {
         return [false, '']
       }
 
-      if (ticker === 'MOBILE') {
-        // If paying with mobile, they need to have enough mobile to cover the payment
-        // and enough hnt to cover the fee
-        const hasEnoughNetwork =
-          networkBalance.minus(paymentState.networkFee).integerBalance >= 0
-        const hasEnoughMobile =
-          mobileSolBalance - paymentState.totalAmount.floatBalance.valueOf() >=
-          0
-        if (!hasEnoughNetwork) return [true, networkBalance.type.ticker]
-        if (!hasEnoughMobile) return [true, 'MOBILE']
-      }
-
-      if (ticker === 'IOT') {
```

Comment:
Dont need this check anymore?

Reply:
No cause IOT only exists in `solana` world